src/index.ts

```diff
@@ -109,9 +118,10 @@ export class I18N {
                 return key;
             }
 
-            result = pluralize(keyValue, count);
+            const pluralizer = this.getLanguagePluralizer(this.lang);
+            result = keyValue[pluralizer(count, PluralForm)] || keyValue[PluralForm.Many];
```

Comment:
Is `PluralForm.Many` valid fallback for all languages? I guess it is responsibility of `pluralizer` to return fallback value?

Reply:
`pluralizer` will always return a `PluralForm`, this is ensured by its type. This is outside of its scope.

This fallback is used when a corresponding i18n string is missing from the keyset. "If we can't use the form recommended by the pluralizer, we will try to use the Many form". I'm not sure this is the correct logic. I see flaws in it. This, however, is how this was implemented earlier, and this is how tests expect this case to be handled. Therefore I propagated this logic to the new version without any changes.

I suggest we discuss this separately and address in another PR.

I will add a clarifying comment to this construction though.

Reply:
Added as a fixup 
When you will be ready to approve, please let me know, so I can squash this into the first commit